acceptance/tests/concurrency/ticket_2659_concurrent_catalog_requests.rb

```diff
@@ -85,7 +85,7 @@
           sleep $sleep_for
           url='https://#{master}:8140/puppet/v3/catalog/#{agent_cert}?environment=production'
           echo "Curling: $url"
-          curl --tlsv1 -v -# -H 'Accept: text/pson' --cert #{cert_path} --key #{key_path} --cacert #{cacert_path} $url
+          env PATH='#{agent[:privatebindir]}:${PATH}' curl --tlsv1 -v -# -H 'Accept: text/pson' --cert #{cert_path} --key #{key_path} --cacert #{cacert_path} $url
```

Comment:
oh right.  thanks josh.  i guess that should stay in beaker and move to a puppet  beaker library later...  only other solution i can think of is specifying the windows and posix paths in options/hosts files and conditional on platform in all_the_tests ick.